### DISCUSSION TITLE
Property 'jsx' does not exist on type 'DetailedHTMLProps<StyleHTMLAttributes<HTMLStyleElement>, HTMLStyleElement>'

### DISCUSSION THREAD

**Discussion by siiddhantt** [teambit/bit#7301] [Help]
I'm trying to use `"styled-jsx": "^5.1.2"` to write CSS inside .tsx files for my Next.js project, and push these components to Bit.

But I'm getting error from compiler on running `bit tag` as : 

![Screenshot from 2023-04-21 23-57-01](https://user-images.githubusercontent.com/47355538/233710232-85ca87ec-adf5-44b4-9377-faa2f96c657b.png)

I'm using styled-jsx like this : 

![Screenshot from 2023-04-21 23-58-29](https://user-images.githubusercontent.com/47355538/233710358-1164fe69-a55e-4374-808f-b7daeb7f0b31.png)

Is it possible to resolve it and export components to Bit?

**Comment by ranm8** [comment]
Seems like you have the types for `styled-jsx` missing. Configure style-jsx in your env.jsonc (https://bit.dev/docs/react-env/dependencies) or use the NextJS environment (https://bit.cloud/frontend/nextjs).
